### ISSUE TITLE
Scrollbar con contrasto insufficiente SC 1.4.11

### ISSUE THREAD

**Issue by SerenaG93** [italia/design-ui-kit#215]
### Esiste già una discussione sul tema che ti interessa, o su un tema simile?

- [x] Ho verificato e non esiste

### Cosa

Le varianti lunghe del componente [Modal](https://www.figma.com/design/OEZ4wXREwlgkQqrshkoghv/UI-Kit-Italia--Community-?node-id=1718-159976) con scroll interno mostrano indicatori della scrollbar con un contrasto insufficiente:

* **Modal / Desktop / Content long:** 1,7:1;
* **Modal / Mobile / Content long:** 1,4:1.

Entrambi i valori sono inferiori al minimo di **3:1** richiesto dal criterio WCAG **1.4.11 Non-text Contrast (AA)**.

<img width="471" height="591" alt="Image" src="https://github.com/user-attachments/assets/f11a29b0-3c45-4928-b8da-e366b477bed8" />
<img width="820" height="717" alt="Image" src="https://github.com/user-attachments/assets/1bb610ce-c5ec-4be7-ae80-b1492bae0870" />

### Perché

Nelle implementazioni in Bootstrap, React e Angular viene usata la scrollbar nativa del browser/sistema operativo. Quindi aspetto e contrasto non dipendono dal componente UI Kit e nelle versioni attualmente implementate non risultano problemi.

Ritengo però utile aggiornare anche il componente in Figma, in modo che sia **accessibile by design** anche nei casi in cui venga utilizzato come unico riferimento per un'implementazione personalizzata della scrollbar.
